manifests/init.pp

```diff
@@ -98,8 +98,34 @@
   $epel_testing_debuginfo_includepkgs     = undef,
   $epel_testing_debuginfo_sslclientkey    = undef,
   $epel_testing_debuginfo_sslclientcert   = undef,
-  Boolean $epel_gpg_managed               = true,
-  $os_maj_release                         = $epel::params::os_maj_release,
+  String $epel_playground_mirrorlist      = "https://mirrors.fedoraproject.org/${url_path}?repo=playground-epel${os_maj_release}&arch=\$basearch",
```

Comment:
for all the urls, can you please use [Stdlib::HTTPSUrl](https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/httpsurl.pp) ?

Reply:
according to https://puppet.com/docs/puppet/5.5/types/yumrepo.html#yumrepo-attribute-mirrorlist we can have absent
so it would be `Variant[Stdlib::HTTPSUrl, Enum['absent']]` ?
We need to add a dependency on stdlib as well then as well (and update metadata.json, .fixtures.yml).
Using `Variant[URI, Enum['absent']` datatype would prevent that additional dependency

Reply:
stdlib is so widely used, so I think depending on it isn't too bad. Please update it to `Variant[Stdlib::HTTPSUrl, Enum['absent']]`

Reply:
Are we sure we don't want to allow `http`?